File: 3. bolum/dowhile/karesi/go/main.go
package main

import "fmt"

func main() {

	var sayi, karesi int
	for {
		fmt.Printf("Lütfen bir sayı giriniz ")
		fmt.Scanf("%d", &sayi)
		karesi = sayi * sayi
		fmt.Printf("Karesi = %d \n", karesi)
		if sayi <= 0 {
			break
		}
	}
	fmt.Println("Program sonlandı")

}

File: 3. bolum/dowhile/karesi/php/main.php
<?php



do {

    $sayi = readline("Lütfen bir sayı giriniz ");
    $karesi = $sayi * $sayi;
    printf("%d Sayini karesi %d", $sayi, $karesi);
} while ($sayi >= 0);
echo "Program sonlandırıldı";
